code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -1435,9 +1435,159 @@ \subsubsection{Data Definitions}
                                                   \item{$γ$ is the soil dry unit weight ($\frac{\text{N}}{\text{m}^{3}}$)}
                                                   \end{symbDescription}
                                                   \\ \midrule \\
-                                                  Source & \cite{fredlund1977}
-                                                           \\ \midrule \\
-                                                           RefBy & 
+                                                  Notes & $b$ is defined in \hyperref[DD:lengthB]{DD: lengthB}.
+                                                          \\ \midrule \\
+                                                          Source & \cite{fredlund1977}
+                                                                   \\ \midrule \\
+                                                                   RefBy & \hyperref[DD:sliceWght]{DD: sliceWght}.
+\\ \bottomrule \end{tabular}
+\end{minipage}
+\par~
+
+\noindent \begin{minipage}{\textwidth}
+\begin{tabular}{p{0.2\textwidth} p{0.73\textwidth}}
+\toprule \textbf{Refname} & \textbf{DD:baseWtrFRDD}
+\phantomsection 
+\label{DD:baseWtrFRDD}
+\\ \midrule \\
+Label & Right base hydrostatic force on a slice
+        \\ \midrule \\
+        Symbol & ${{U^{R}}_{b}}$
+                 \\ \midrule \\
+                 Units & $\frac{\text{N}}{\text{m}}$
+                         \\ \midrule \\
+                         Equation & \begin{displaymath}
+                                    {{U^{R}}_{b}}={ℓ_{b,i}} \begin{cases}
+\left({y_{wt,i}}-{y_{slip,i}}\right) {γ_{w}}, & {y_{wt,i}}>{y_{slip,i}}\\
+0, & {y_{wt,i}}\leq{}{y_{slip,i}}
+\end{cases}
+                                    \end{displaymath}
+                                    \\ \midrule \\
+                                    Description & \begin{symbDescription}
+                                                  \item{${{U^{R}}_{b}}$ is the right base hydrostatic force on a slice ($\frac{\text{N}}{\text{m}}$)}
+                                                  \item{${ℓ_{b}}$ is the total base length of a slice (m)}
+                                                  \item{$i$ is the index (Unitless)}
+                                                  \item{${y_{wt}}$ is the y ordinate (m)}
+                                                  \item{${y_{slip}}$ is the y ordinate (m)}
```

Comment:
I am going to create a separate issue about the use of the term y ordinate.  I think we should just say y coordinate.  y ordinate is not a term that is used, as far as I can tell.  ordinate is used as a name of the vertical axis, but y ordinate is redundant.  I do not want to confuse this pull request with this issue, so I'll make the issue a separate one.